sale_mail_autosubscribe/data/mail_autosubscribe.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
do we really need a module only for this? :thinking: 

Looks like proper documentation in README of the base module is enough.

Reply:
I asked myself the same question but then I decided to add the module because this way it's easier for non-functional people.

Imagine people installing the module from the odoo appstore, for example

Reply:
fair enough :)